go.mod

```diff
@@ -4,19 +4,19 @@ go 1.18
 
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/NVIDIA/k8s-operator-libs v0.0.0-20230118182127-8e73980389ca
```

Comment:
nit, just making sure `go mod tidy` was run.

Reply:
Yes, it was. Do you have a concern about something here?